code/script.js

```diff
@@ -1,22 +1,28 @@
-// All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
-
-// Global variables, if you need any, declared here
 
-// Functions declared here
+const chat = document.getElementById('chat')
+const handleNameInput = document.getElementById('name-input')
+const sendBtn = document.getElementById('send-btn')
+const nameForm = document.getElementById('name-form')
+const inputWrapper = document.getElementById('input-wrapper')
+const userSound = document.getElementById('clickUser')
+let questionNumber = 1
 
-// This function will add a chat bubble in the correct place based on who the sender is
+//Decides who says what and puts the bubbles in the chat 
 const showMessage = (message, sender) => {
   if (sender === 'user') {
+    let sound = new Audio("assets/userClick.wav")
+    sound.play();
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/user.png" alt="User" />
       </section>
     `
   } else if (sender === 'bot') {
+    let sound = new Audio("assets/botSound.wav")
```

Comment:
Using different sounds for the bot and the user is a great idea in terms of accessibility! People who guide themselves by sound will appreciate this feature a lot :)